app/src/main/java/cellscanner/wowtor/github/com/cellscanner/recorder/Recorder.java

```diff
@@ -0,0 +1,27 @@
+package cellscanner.wowtor.github.com.cellscanner.recorder;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+
+/**
+ * Responsible for controlling the recording functionality
+ */
+public class Recorder {
```

Comment:
It is not actually a recorder; maybe pick a name that fits its purpose, like `Preferences`?

Reply:
In future change should start the correct service. Picked name in advance